.eslintrc.json

```diff
@@ -14,7 +14,12 @@
     "no-param-reassign": "off",
     "eol-last": "off",
     "arrow-parens": "off",
-    "no-descending-specificity": "off"
+    "no-descending-specificity": "off",
+    "no-use-before-define":"off",
+    "no-plusplus":"off",
+    "default-case":"off",
+    "no-unused-vars": "off",
+    "import/prefer-default-export": "off"
```

Comment:
Opps, we have some little changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/javascript)
![image](https://user-images.githubusercontent.com/31897434/73417568-ca44bd00-4318-11ea-8a5e-fedabd629b8c.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/


If worst case you need to set a new rule, use inline comment;
https://eslint.org/docs/rules/no-inline-comments